content/en/tracing/guide/add_span_md_and_graph_it.md

```diff
@@ -92,10 +92,8 @@ require 'ddtrace'
 class ShoppingCartController < ApplicationController
   # GET /shopping_cart
   def index
-    # Get the active span
-    current_span = Datadog.tracer.active_span
-    # customer_id -> 254889
-    current_span.set_tag('customer.id', params.permit([:customer_id])) unless current_span.nil?
+    # Get the active span and set customer_id -> 254889
+    Datadog::Tracing.active_span&.set_tag('customer.id', params.permit([:customer_id]))
```

Comment:
![6eahco](https://user-images.githubusercontent.com/2785847/165915691-6e0eb5da-7028-4478-8e3d-eb8cd13ace2a.jpeg)

Reply:
Also @lloeki  would it make sense to change this doc once https://github.com/DataDog/dd-trace-rb/pull/1954 is merged in? (If so, `content/en/tracing/setup_overview/custom_instrumentation/ruby.md` has a similar example as well)

Reply:
These tags seem different: `usr.*` tags have a specific semantic that is going to be processed backed-side.

https://github.com/DataDog/documentation/blob/eae1fd30edf42e1feb780aaecab6fcd2a4ac1b0b/content/en/logs/log_configuration/attributes_naming_convention.md#user-related-attributes

There's a specific doc for`set_user` (or if not implemented, `set_tag` with expected tags) but I can't seem to find it.

Reply:
> specific doc

It's [here](https://github.com/DataDog/documentation/blob/eae1fd30edf42e1feb780aaecab6fcd2a4ac1b0b/content/en/security_platform/application_security/setup_and_configure.md). There's no anchor so search for `trace.set_tag`.

Reply:
Do you think it's worth switching this example to the other tag name, or just leave it be?